src/fontman/font_manager.cpp

```diff
@@ -575,39 +575,53 @@ void FontManager::printText(const char* text, size_t text_size,
                             int x, int y,
                             int font,
                             float Red, float Green, float Blue, float Alpha,
-                            uint32_t ttf_FontSize)
+                            uint32_t ttf_FontSize, bool outline)
```

Comment:
Maybe you'll also provide an option to specify a colour?

Reply:
Great idea, done.